### PR TITLE
Added a section on pod stability 

### PR DESCRIPTION
**Pull request by alexmilowski**
section on thresholds, eviction, monitoring pressure, quality of service class, and assigning priority

### CODE REVIEW

File: content/platforms/kubernetes/stability.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
```
Comment (amiramm) [inline content/platforms/kubernetes/stability.md:1]: @alexmilowski Please also see the first few topics here: https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/topics.md

Comment (alexmilowski) [reply content/platforms/kubernetes/stability.md:1]: That document says the same things so we're not contradicting ourselves. Everything that is after [PSP](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/topics.md#pod-security-policy-psp) needs to go into a different section. I'll do that via a different PR.

**Comment by bmansheim** [comment]
@alexmilowski Please review my edits.

File: content/platforms/kubernetes/stability.md [hunk 2]
```diff
@@ -1,53 +1,46 @@
...
-## Guaranteeing quality of service
+At pod creations, Kubernetes assigns the pod one of three [quality of service classes](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/):
+Guaranteed, Burstable, and Best Effort.
+You can assign the Guaranteed class to the Redis Enterprise node pods.
 
-When Kubernetes creates a pod it assigns it one of three [quality of service
-classes](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/):
-Guaranteed, Burstable, and Best Effort. For Redis Enterprise node
-pods we want all the pods to be assigned the Guaranteed class.
+To assign a quality of service class:
```
Comment (alexmilowski) [inline content/platforms/kubernetes/stability.md:19]: We don't assign the quality of service - K8s does.

File: content/platforms/kubernetes/stability.md [hunk 3]
```diff
@@ -1,53 +1,46 @@
...
 
-To do so, each operator managed pods must have:
+ * Every container in the pod must have a memory limit and a memory request, and they must be the same.
+ * Every container in the pod must have a CPU limit and a CPU request, and they must be the same.
```
Comment (alexmilowski) [inline content/platforms/kubernetes/stability.md:22]: This is fine.

File: content/platforms/kubernetes/stability.md [hunk 4]
```diff
@@ -1,53 +1,46 @@
...
- controlling this behaviour.
+description: You can use quality of service, priority class, eviction thresholds and resource monitoring
+    to maintain cluster node pod stability.
 weight: 90
 alwaysopen: false
 categories: ["Platforms"]
 aliases:
 ---
+Kubernetes clusters manage the allocation of system resources and can evict pods to release system resources.
+Here are some ways that you can configure the Redis Enterprise node pods to maintain pod stability:
```
Comment (alexmilowski) [inline content/platforms/kubernetes/stability.md:11]: Okay.

File: content/platforms/kubernetes/stability.md [hunk 5]
```diff
@@ -1,53 +1,46 @@
...
-pods we want all the pods to be assigned the Guaranteed class.
+To assign a quality of service class:
 
-To do so, each operator managed pods must have:
+ * Every container in the pod must have a memory limit and a memory request, and they must be the same.
+ * Every container in the pod must have a CPU limit and a CPU request, and they must be the same.
 
- * Every Container must have a memory limit and a memory request, and they must be the same.
- * Every Container must have a CPU limit and a CPU request, and they must be the same.
+These requirements are met in the default version of your Redis Enterprise cluster CRD,
```
Comment (alexmilowski) [inline content/platforms/kubernetes/stability.md:24]: This is not what happens. The CRD does not contain requests and limits unless you put them in there.

File: content/platforms/kubernetes/stability.md (hunk 2 above)
Comment (alexmilowski) [reply content/platforms/kubernetes/stability.md:19]: I updated this.

File: content/platforms/kubernetes/stability.md (hunk 5 above)
Comment (alexmilowski) [reply content/platforms/kubernetes/stability.md:24]: I fixed this.

File: content/platforms/kubernetes/stability.md [hunk 6]
```diff
@@ -1,53 +1,46 @@
...
+If you assign a high priority value, the deployment of other applications cannot preempt a Redis Enterprise node pod.
 
-When a Redis Enterprise node pod is scheduled it can be assigned a [
-priority class](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/)
-via the `priorityClassName` property.  In this approach, we create a sufficiently
-large priority that prevents the deployment of other applications from causing
-a Redis Enterprise node pod from being preempted.
-
-The class must be first defined for the cluster with a sufficiently very large value:
+You must assign a priority class for the cluster with a high value:
```
[the remark this answers is not in the document]
Comment (alexmilowski) [reply content/platforms/kubernetes/stability.md:43]: I have fixed this.

**Comment by alexmilowski** [comment]
I have reviewed the style changes and made a few updates.

**Pull request merged**